### PR TITLE
KSH-8 : Linear gradient background for the app

### PR DESCRIPTION
**Pull request by pandyama**
Linear - https://linear.app/oddhours/issue/KSH-8/homescreen-landing-page

Tests
- run `npm install` to get the `expo-linear-gradient` package installed on your local repo
- run `expo start` & verify on your device that the gradient looks good

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
 
-export default function App() {
+export default function App () {
   return (
-    <View style={styles.container}>
-      <Text>Open up App.tsx to start working on your app!</Text>
-      <StatusBar style="auto" />
-    </View>
-  );
+    <LinearGradient
```
Comment (oyiza) [inline App.tsx:10]: is it possible to put the linear gradient style definition in our baseStyle.tsx? if it's not possible that's also alright

**Review by oyiza — APPROVED** [review]

File: App.tsx (hunk 1 above)
Comment (pandyama) [reply App.tsx:10]: @oyiza which style definition, can you point out the line of code?

Comment (oyiza) [reply App.tsx:10]: actually nevermind. i was referring to the entire linearGradient component but the style is already in the styles file

Comment (pandyama) [reply App.tsx:10]: yup!

**Comment by pandyama** [comment]
@oyiza once we get the fonts Ticket done, we can create a prod PR, what do you say?

**Comment by oyiza** [comment]
@pandyama ya. depends on how that goes actually. i'm having some difficulty but we'll see

**Comment by pandyama** [comment]
yaa font setup is a little bit of a bitch... i did it over the weekend for a separate project that I am doing so im using that project as the test environment for keepshift lol... i'll send you over what resources i used later tonight

**Comment by oyiza** [comment]
lol okay sounds good. ya it's not responding for whatever reason

**Pull request merged**